src/Common/utils.ts

```diff
@@ -5,7 +5,7 @@ import * as addressCodec from 'ripple-address-codec'
 /**
  * A prefix applied when hashing a signed transaction blob.
  *
- * @see https://xrpl.org/basic-data-types.html#hashes
+ * @see {@link https://xrpl.org/basic-data-types.html#hashes|XRPL Data Types - Hashes}.
```

Comment:
Again, TSDoc likes to pair `@see` tags with `@link` tags to clarify what is going on.